curriculum/challenges/english/25-front-end-development/quiz-python-basics/67f41242431cbf3db8ca79c7.md

```diff
@@ -17,439 +17,546 @@ To pass the quiz, you must correctly answer at least 18 of the 20 questions belo
 
 #### --text--
 
-Placeholder question
+Which of the following functions is used to check if a variable matches a specific data type?
 
 #### --distractors--
 
-Placeholder distractor 1
+`istype()`
 
 ---
 
-Placeholder distractor 2
+`isdata()`
 
 ---
 
-Placeholder distractor 3
+`isvariable()`
 
 #### --answer--
 
-Placeholder answer
+`isinstance()`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following is NOT a form of string concatenation?
 
 #### --distractors--
 
-Placeholder distractor 1
+```py
+greeting = 'My name is '
+developer = 'Jessica'
+
+greeting += developer
+```
 
 ---
 
-Placeholder distractor 2
+```py
+developer = 'Jessica'
+greeting = 'My name is ' + developer + '.'
+```
 
 ---
 
-Placeholder distractor 3
+```py
+developer = 'Jessica'
+age = 30
+greeting = 'My name is ' + developer + ' and I am ' + str(age) + ' years old.'
+```
 
 #### --answer--
 
-Placeholder answer
+```py
+developer = 'Jessica'
+greeting = x'My name is {developer}.'
+```
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following methods is used to return the number of the characters in the string?
 
 #### --distractors--
 
-Placeholder distractor 1
+```py
+counting()
+```
 
 ---
 
-Placeholder distractor 2
+```py
+length()
+```
 
 ---
 
-Placeholder distractor 3
+```py
+iscount()
+```
 
 #### --answer--
 
-Placeholder answer
+```py
+len()
+```
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What will `result` be in this example?
+
+```py
+developer = 'Naomi'
+
+result = developer.endswith('N') # ?
+```
 
 #### --distractors--
 
-Placeholder distractor 1
+`Undefined`
 
 ---
 
-Placeholder distractor 2
+`True`
 
 ---
 
-Placeholder distractor 3
+`None`
 
 #### --answer--
 
-Placeholder answer
+`False`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What happens when you add a float and an integer?
 
 #### --distractors--
 
-Placeholder distractor 1
+The result will be an error message.
 
 ---
 
-Placeholder distractor 2
+The result will be None.
 
 ---
 
-Placeholder distractor 3
+The result will be an integer.
 
 #### --answer--
 
-Placeholder answer
+The result will be a float.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following is the correct way to define a function?
 
 #### --distractors--
 
-Placeholder distractor 1
+```py
+define get_sum(num_1, num_2) {
+  return num_1 + num_2 
+} 
+```
 
 ---
 
-Placeholder distractor 2
+```py
+set get_sum(num_1, num_2):
+  return num_1 + num_2
+```
 
 ---
 
-Placeholder distractor 3
+```py
+function get_sum(num_1, num_2) {
+  return num_1 + num_2 
+}
+```
 
 #### --answer--
 
-Placeholder answer
+```py
+def get_sum(num_1, num_2):
+  return num_1 + num_2
+```
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following is an example of a multi line comment?
```

Comment:
Technically speaking Python doesn't have multi-line comments. In case of multi-line comments, each line should start with the _standard_ `#` - https://peps.python.org/pep-0008/#block-comments.

With triple quotes are either docstrings or multi-line strings - in both cases contents need to be valid string.